packages/ucp/docs/getting-started.md

```diff
@@ -0,0 +1,361 @@
+This tutorial is about how to install and use the [@solidlab/ucp](https://github.com/SolidLabResearch/user-managed-access/tree/main/packages/ucp) library utilities.
+
+## Installing
+
+As of today (2 Februari 2024), this package is not on npm.
+Which means installing via `npm install @solidlab/ucp` does **not** work yet.
+
+However, it can still easily installed by adding `"@solidlab/ucp": "https://github.com/woutslabbinck/ucp-enforcement.git"` to your dependencies in package.json.
```

Comment:
Updated to use GitPkg, so the installations keep up to date with the code in this repo.

```suggestion
However, it can still easily installed by adding `"@solidlab/ucp": "https://gitpkg.now.sh/SolidLabResearch/user-managed-access/packages/css?main"` to your dependencies in package.json, or executing the corresponding command of your favourite package manager, e.g. `yarn add @solidlab/ucp@"https://gitpkg.now.sh/SolidLabResearch/user-managed-access/packages/css?main"`.
```